inspirehep/modules/workflows/workflows/core_selection.py

```diff
@@ -41,6 +41,7 @@ class CoreSelection(object):
             message='Submission halted Waiting for curator to decide if record is CORE.'
         ),
         load_record_from_hep,
+        store_head_version,
```

Comment:
Now the question is do we want this to be loaded like that? load_record_from_hep is loading record through REST API, maybe it will be just better to update `load_record_from_hep` to load it with the version?

Reply:
Since we have it like that in many places, I would just leave it the same. We should cleanup many parts in workflows

Reply:
Can we create task to stop reading records from DB in workflows and read them only form HEP?

Reply:
I will create a taks